test/modules/generic-mqtt-tester/src/message_channel.rs

```diff
@@ -141,7 +149,6 @@ where
             match future::select(received_pub, shutdown_signal).await {
                 Either::Left((received_publication, _)) => {
                     if let Some(received_publication) = received_publication {
-                        info!("received publication {:?}", received_publication);
```

Comment:
We can remove this log since it is also getting logged when received by mqtt3 client.